packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js

```diff
@@ -220,6 +220,20 @@ QUnit.module('options changing', {
         }
     });
 
+    QUnit.test('The validation error should be null after removing the mask (T1214604)', function(assert) {
+        const textBox = $('#textbox').dxTextBox({
+            value: '123',
+            mask: '000000/0009',
+            maskInvalidMessage: 'mask error',
+        }).dxTextBox('instance');
```

Comment:
This test is incorrect at the moment. You can see it because passed ` maskInvalidMessage: 'mask error',` config is not applied and error is still `'Value is invalid'`.  It's because u initiate new textbox on `$('#textbox')` element but some instance is already attached to it in `beforeEach`

Reply:
[c0783df5eff72f2e9d00601f6521ea5721a42bf0]